pipelines/broad/qc/CheckFingerprint.documentation.md

```diff
@@ -0,0 +1,62 @@
+
+** NOTE that if neither input_vcf nor input_bam is provided, CheckFingerprint will return the fingerprint as pulled from Mercury, or supplied as input.
+
+### Fingerprint Genotypes
+*    read_fingerprint_from_mercury. If true, the CheckFingerprint pipeline will attempt to retrieve fingerprint genotypes from the Mercury Fingerprint Store. If false, an input fingerprint genotypes file (VCF) is required.
+* fingerprint_genotypes_vcf [Optional]. A VCF file containing fingerprint genotypes. Required if you are not retrieving fingerprint genotypes from Mercury.
+*    fingerprint_genotypes_vcf_index [Optional]. The index of the fingerprint genotypes VCF.
+*    sample_lsid [Optional]. The sample lsid used to retrieve the fingerprint genotypes from Mercury. This parameter is required if 'read_fingerprint_from_mercury' is true.
+*    sample_alias. The sample alias for the fingerprint genotypes file. If 'read_fingerprint_from_mercury' is true this is the value that is put into the fingerprint genotypes file generated when pulling the fingerprints from Mercury. If 'read_fingerprint_from_mercury' is false AND there is more than one sample in the fingerprint_genotypes_vcf, this parameter lets CheckFingerprint know which sample in the fingerprint_genotypes_vcf to use.
+
+### Reference
+*   ref_fasta. The cloud path of the reference FASTA used.
+*   ref_fasta_index. The cloud path of the index file for the reference FASTA.
+*   ref_fasta_dict. The cloud path of the dictionary file for the reference FASTA.
+*   haplotype_database_file. The cloud path of the ‘haplotype_database_file’. This is a file that contains the haplotype block information for the fingerprinting sites.
+
+### Other
+* output_base_name. A string used as the base name for the metrics file generated by CheckFingerprint.
+*   environment [optional].  A string value indicating which environment (dev, or prod) this WDL is to run on. This parameter is required if 'read_fingerprint_from_mercury' is true.
+*   vault_token_path [optional]. The cloud path to the vault token. This parameter is required if 'read_fingerprint_from_mercury' is true.
+
+
+## Pipeline Outputs
+
+The pipeline generates the following outputs:
+
+*   fingerprint_read_from_mercury. If the parameter 'read_fingerprint_from_mercury' is true, the CheckFingerprint pipeline will attempt to read the fingerprint from Mercury using the provided sample_lsid. If the pipeline is successful (the sample lsid is valid and fingerprint genotypes are read from Mercury), then this value will be true.
+*   reference_fingerprint_vcf. The fingerprint genotypes VCF. This can be the result of reading the fingerprints from Mercury (if 'read_fingerprint_from_mercury' is true), or the fingerprint_genotypes_vcf that was provided as input. It is possible for this to be undefined if there the pipeline is unable to read fingerprints from Mercury (in which case the output `fingerprint_read_from_mercury' will be false).
```

Comment:
```suggestion
*   reference_fingerprint_vcf. The fingerprint genotypes VCF. This can be the result of reading the fingerprints from Mercury (if 'read_fingerprint_from_mercury' is true), or the fingerprint_genotypes_vcf that was provided as input. It is possible for this to be undefined if the pipeline is unable to read fingerprints from Mercury (in which case the output `fingerprint_read_from_mercury' will be false).
```

Reply:
This has been corrected